return {
	"[v2.6.2 - 2024-08-30]";
	"Added !sit command for donors";
	"Several backend improvements";
	"[v2.6.0c - 2024-08-29]";
	"Several backend improvements";
	"[v2.6.0b - 2024-08-22]";
	"Several backend improvements";
	"Fixed ;sword";
	"[v2.6.0a - 2024-08-09]";
	"Several backend improvements";
	"Fixed UserPanel scaling issues";
	"[v2.6.0 - 2024-08-05]";
	"Whitelist and serverlock now use join filters";
	"New smartlog with many improvements";
	"Count command download button";
	"Damagelogs now work again";
	"!Mypoints command to view your points";
	"You can now enable chat UI tabs with !client";
	"Serverinfo should now always show the server location (moderator+ can refresh the server location with ;refreshserverlocation)";
	"!Hastools playerfinder - returns any player that has a tool";
	";Invisible and ;visible now correctly set your current weapons transparency to invisible and back to the right transparency.";
	"";
	"[v2.5.9c - 2024-08-02]";
	"Made a temporary fix for command errors not appearing";
	"Did some backend performance tweaks";
	"";
	"[v2.5.9b - 2024-07-27]";
	"Several backend improvements and fixes";
	"Fixed ;view not working";
	"Fixed permission levels for several commands";
	"";
	"[v2.5.9a - 2024-07-25]";
	"Several backend improvements";
	"";
	"[v2.5.9 - 2024-07-15]";
	"Security fixes";
	"Hopefully fixed chat tabs and displayname disabling";
	"Fixed RunCommand";
	"Admins can now opt out of receiving help messages (!help)";
	"Admins can now disable and remove donor effects";
	"";
	"[v2.5.7 - 2024-06-02]";
	"Some security fixes";
	"";
	"[v2.5.6 - 2024-05-27]";
	"KOS system similar to PBOP";
	"Collision related commands were fixed";
	"Security Improvements";
	"";
	"[v2.5.5 - 2024-04-14]";
	"Unfreeze has been fixed... Finally...";
	"A lot of new things for moderators";
	"";
	"[v2.5.4 - 2024-03-24]";
	"More placeholder texts around the userpanel";
	"Keybind set fix";
	"Refresh now gives the player back flight and god (can be skipped)";
	"New icons (again)";
	"";
	"[v2.5.3 - 2024-03-17]";
	"Serverlock duplicate message fix";
	"New refresh icon with a smoother animation";
	"Aliases should now be hidden in chat if HideChatCommands is enabled";
	"Added a client setting to toggle lists being selectable";
	"";
	"[v2.5.2B - 2024-02-14 12:00 UTC] - The UI update - @EasternBloxxer";
	"*My own implementation of round TabButtons. This should have way less issues than the last system from around 2018.....";
	"*Console  changes";
	"*When ui is hidden it should no longer appear white. - this was due to the UIGrandient not existing in Drag";
	"";
	"[v2.5.2A - 2024-02-09] @EasternBloxxer";
	"*Whitelist has been fixed";
	"*Prints not use print instead of TestService:Message this is because it cannot  correctly print tables.";
	"";
	"[v2.5.2 - 2024-02-03] @EasternBloxxer";
	"+Implemented whitelist list";
	"+You can now choose to kick any nonadmins when locking a server";
	"*A lot of the ARIDe UI is now Blue";
	"*Place command has been fixed";
	"*Command logs are now correctly sent to BinaryBridge server";
	"*Optimized ARIDe's bytecode interpreter";
	"*Disabled debug mode in live builds";
	"";
}
